packages/elements/src/tree/elements/tree-item.ts

```diff
@@ -71,7 +71,7 @@ export class TreeItem<T extends TreeDataItem = TreeDataItem> extends ControlElem
    * Icon of the item
    */
   @property({ type: String })
-  public icon = '';
+  public icon: string | undefined = '';
```

Comment:
Why it can be undefined?

Reply:
Tree renderer set `icon` to `undefined` so Tree Item could be rendered it correctly. [Tree Item also expect that  `icon` could be `undefined`](https://github.com/Refinitiv/refinitiv-ui/blob/6f8b53351774c474b708664e75d326954cf1b231/packages/elements/src/tree/elements/tree-item.ts#L138).

Reply:
It should change to this, and the `createTreeRenderer` no need to cast to string
```suggestion
  public icon?: string = '';
```

Reply:
The casting was unnecessary, good catch! Adopted your suggestion.